ddl/index.go

```diff
@@ -1411,16 +1411,24 @@ func (w *worker) addPhysicalTableIndex(t table.PhysicalTable, indexInfo *model.I
 	logutil.BgLogger().Info("[ddl] start to add table index", zap.String("job", job.String()), zap.String("reorgInfo", reorgInfo.String()))
 	totalAddedCount := job.GetRowCount()
 
+	// It is used to close #19325 in earlier versions.
```

Comment:
I think we can remove this comment, or use a more understandable comment. Can we expect the reader to open github and search for this issue?

Reply:
ok